x-pack/plugins/upgrade_assistant/public/application/components/overview/fix_logs_step/fix_logs_step.tsx

```diff
@@ -134,10 +145,18 @@ const FixLogsStep: FunctionComponent<Props> = ({
       </EuiText>
       <EuiSpacer size="m" />
       <EuiPanel>
-        <DeprecationLoggingToggle {...state} />
```

Comment:
By passing through the entire object returned by `useDeprecationLogging` we create a tight coupling between that hook's return value and `DeprecationLoggingToggle` props, which creates brittleness if we make changes to either one.